Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/index.adoc

= Lisk Elements Overview
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Elements overview gives an introduction to the Lisk Elements libray and it's different sub-packages. It describes the basic usage and how to upgrade with NPM.

Reviewer: ```suggestion
:description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here..
```
